docs/user/site.rst

```diff
@@ -21,7 +21,7 @@ site_code
     The code of your community. It is good practice to use the TLD of
     your community here.
 
-domain_seed
```

Comment:
Has this been changed? The release notes mention that there are no changes since 2018.1.x

Reply:
Thanks, this was obviously an accidental change, fixed.